client/src/pages/scoreboard.js

```diff
@@ -5,141 +5,157 @@ import Pagination from '../components/pagination'
 
 import { getScoreboard } from '../api/scoreboard'
 import { privateProfile } from '../api/profile'
+import { Component, createRef } from 'preact';
 
 const PAGESIZE_OPTIONS = [25, 50, 100]
 
 const Scoreboard = withStyles({
   frame: {
     paddingBottom: '10px'
+  },
+  selected: {
+    backgroundColor: "#fdd"
   }
-}, ({ classes }) => {
-  const loggedIn = useMemo(() => localStorage.getItem('token') !== null, [])
-  const [profile, setProfile] = useState(null)
-  const [pageSize, _setPageSize] = useState(100)
-  const [scores, setScores] = useState([])
-  const [division, _setDivision] = useState('')
-  const [page, setPage] = useState(1)
-  const [totalItems, setTotalItems] = useState(0)
+}, class Scoreboard extends Component {
```

Comment:
This is mixing class components with hooks, please keep this a function component.